src/math/Vector4.js

```diff
@@ -315,17 +315,17 @@ Object.assign( Vector4.prototype, {
 			m31 = te[ 2 ], m32 = te[ 6 ], m33 = te[ 10 ];
 
 		if ( ( Math.abs( m12 - m21 ) < epsilon ) &&
-		     ( Math.abs( m13 - m31 ) < epsilon ) &&
-		     ( Math.abs( m23 - m32 ) < epsilon ) ) {
+			 ( Math.abs( m13 - m31 ) < epsilon ) &&
+			 ( Math.abs( m23 - m32 ) < epsilon ) ) {
 
 			// singularity found
 			// first check for identity matrix which must have +1 for all terms
 			// in leading diagonal and zero in other terms
 
 			if ( ( Math.abs( m12 + m21 ) < epsilon2 ) &&
-			     ( Math.abs( m13 + m31 ) < epsilon2 ) &&
-			     ( Math.abs( m23 + m32 ) < epsilon2 ) &&
-			     ( Math.abs( m11 + m22 + m33 - 3 ) < epsilon2 ) ) {
+				 ( Math.abs( m13 + m31 ) < epsilon2 ) &&
+				 ( Math.abs( m23 + m32 ) < epsilon2 ) &&
+				 ( Math.abs( m11 + m22 + m33 - 3 ) < epsilon2 ) ) {
```

Comment:
Now these lines are harder to read 😕

Reply:
> Now these lines are harder to read 😕

I am not sure other editor what look like it, But in VSC is same.
| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/23699926/89106271-1ce36b00-d45b-11ea-9702-1c1e6e304f6a.png) | ![image](https://user-images.githubusercontent.com/23699926/89106255-f0c7ea00-d45a-11ea-8129-8ee751acd36a.png) |

Reply:
Try changing the tab size in VSC.